### ISSUE TITLE
Font size should be selectable for better accessibility

### ISSUE THREAD

**Issue by timceesq** [hfiref0x/WinDepends#43]
The font size used in the GUI should be selectable somewhere in the options/configuration dialog from a drop-down list of sizes. This is not to be confused with the overall size/scaling/DPI of the whole GUI and its graphical elements. Just the font size, and no other size of anything else, should be explicitly selectable from a drop-down list/combobox, as per this feature request.

Ideally, the whole font face, along with its size, should be selectable (in which case the Windows' built-in font selection dialog should be used, not a custom one).

**Additional context**
Explicit font size change is required for persons with accessibility issues. This shouldn't require a great deal of work which is also a plus.

**Comment by hfiref0x** [comment]
Font size is now adjustable (see Configuration dialog), but the font face remains fixed to ensure UI consistency and layout stability.
